app/templates/buyers/_base_edit_question_page.html

```diff
@@ -7,24 +7,32 @@
 
 {% block mainContent %}
 
+  {% set is_govuk_frontend = govuk_frontend_from_question(question) %}
```

Comment:
So at the moment, this will only be truthy if `question` is type `text`, correct?

Reply:
Correct